src/utils.js

```diff
@@ -21,3 +21,15 @@ export const fetchTopics = () => {
     return res.data.topics;
   });
 };
+
+export const fetchArticlesById = article_id => {
+  return myApi.get(`/articles/${article_id}`).then(res => {
+    return res.data.article;
+  });
+};
+
+export const fetchComments = article_id => {
```

Comment:
Sorry will need to get rid of this api call and save it for the comments ticket 🙂